setup.py

```diff
@@ -69,6 +69,8 @@
     # Needed for packaging
     #
     "wheel",
+    # Needed to deploy from web mode
+    "requests",
```

Comment:
Do we need a min version here?

Reply:
Probably not, but I'll include one for completeness.

Reply:
Great 👍, let's just make sure it's not too new if it's not neccesary, so that older versions from OS package managers can satisfy it.